src/neptune_pytorch/impl/__init__.py

```diff
@@ -154,9 +155,13 @@ def hook(module, input, output):
                 dot = torchviz.make_dot(output, params=dict(module.named_parameters()))
                 # Use tempfile correctly.
                 dot.format = "png"
-                dot.render(outfile="torch-viz.png")
-                self._namespace_handler["model"]["visualization"].upload("torch-viz.png")
-                self._is_viz_saved = True
+                try:
+                    dot.render(outfile="torch-viz.png")
+                    self._namespace_handler["model"]["visualization"].upload("torch-viz.png")
+                except ExecutableNotFound:
```

Comment:
If native package for rendering `dot` is not installed then `graphviz` throws this error.